experimental/material_3_demo/lib/component_screen.dart

```diff
@@ -2291,3 +2386,26 @@ class ComponentGroupDecoration extends StatelessWidget {
     );
   }
 }
+
+enum ColorItem {
```

Comment:
Presumable everything still works if there are more colors than can fit?

Reply:
Yes, then the result should be able to scroll.